Multisig/Data/Services/Safe Client Gateway Service/TransactionDetailsRequest.swift

```diff
@@ -10,23 +10,26 @@ import Foundation
 
 struct TransactionDetailsRequest: JSONRequest {
     var id: String
+    let chainId: Int
+    
     var httpMethod: String { "GET" }
-    var urlPath: String { "/v1/transactions/\(id)" }
+    var urlPath: String { "/\(chainId)/v1/transactions/\(id)" }
     typealias ResponseType = SCGModels.TransactionDetails
 }
 
 extension TransactionDetailsRequest {
-    init(safeTxHash: Data) {
+    init(safeTxHash: Data, chainId: Int) {
         id = safeTxHash.toHexStringWithPrefix()
+        self.chainId = chainId
     }
 }
 
 extension SafeClientGatewayService {
```

Comment:
@MouazAlzahabi did you run integration tests? I think client gateway tests will break now